lib/src/modules/settings/scenes/flutter_settings.scene.dart

```diff
@@ -49,20 +50,20 @@ class SettingsSectionFlutter extends HookWidget {
                             width: 0.5,
                           ),
                         ),
-                        child: const Text(
-                          'A Flutter sdk version neeeds to be set as global '
-                          'in order to access Flutter settings',
+                        child: Text(
+                          S.of(context).aFlutterSdkVersionNeeedsToBeSetAsGlobal +
+                          S.of(context).inOrderToAccessFlutterSettings,
                         ),
                       ),
                       const SizedBox(height: 20)
                     ],
                   ),
             SwitchListTile(
-              title: const Text('Analytics & Crash Reporting'),
-              subtitle: const Text(
-                'When a flutter command crashes it attempts'
-                ' to send a crash report to Google in order to help'
-                ' Google contribute improvements to Flutter over time',
+              title: Text(S.of(context).analyticsCrashReporting),
+              subtitle: Text(
+                S.of(context).whenAFlutterCommandCrashesItAttempts +
```

Comment:
All these 3 lines can be added to the same string as the space was just for readibility in the code